### ISSUE TITLE
1.19.4 BungeeControl-Red Cannot execute FORWARD_COMMAND

### ISSUE THREAD

**Issue by x1p** [kangarko/ChatControl#2338]
### "/version ChatControlRed" - plugin version

ChatControlRed version 10.17.8 up to 10.18.2

### Are you using MySQL?

Yes

### Are you using BungeeCord?

Yes

### Information about the issue/bug

When a player swears, points are given etc, the settings file can execute something like:
```yaml
Warning_Points:
  Sets:
    swear:
      2:
      - 'bungeeconsole bungee tempmute {player} 5m Swearing is prohibited
```
BungeeControl-Red then gives this error and the command is not executed
```
[BungeeControl-Red] Cannot execute FORWARD_COMMAND because the sender of the command (PlayerABC123) lacks 'chatcontrol.command.forward' permission.
```
I can't remember having to give the player this permission as this command should be executed by the console.

**Comment by kangarko** [comment]
This is a new protection measure against malicious clients. You need to install a permission plugin on Bungee and explicitly give admins permissions to execute commands.

**Comment by x1p** [comment]
Why would I give admins this permission? There is no admin online. I never said this is executed by an admin.
It is executed by CHC on the console of the paper server that player is present to be executed on the bungee by CHC. So how can I give the console the permission?

- Player swears.
- Receives points.
- Points threshold is reached. CHC executes a bungee command `as console` that auto-mutes the swearing player.

The only player online in this case is the player who swears. So you imply that a player should have this permission.
If so, that would bypass any security measures imho.

TL:DR; so every player should have the chatcontrol.command.forward & chatcontrol.command.bungee permission?

**Comment by kangarko** [comment]
I see, there are some problems with it as you pointed out, I am going to have a deeper look.

**Comment by kangarko** [comment]
I have temporarily lifted the bungee perm requirement and conducting more research. A new version will come out tonight.
